pkg/kv/bulk/sst_batcher.go

```diff
@@ -380,8 +380,12 @@ func (b *SSTBatcher) doFlush(ctx context.Context, reason int, nextKey roachpb.Ke
 				} else {
 					b.flushCounts.splitWait += reply.Timing.Split
 					b.flushCounts.scatterWait += reply.Timing.Scatter
-					if reply.Stats != nil {
-						b.flushCounts.scatterMoved += sz(reply.Stats.Total())
+					if reply.ScatteredStats != nil {
+						moved := sz(reply.ScatteredStats.Total())
+						b.flushCounts.scatterMoved += moved
+						if moved > 0 {
+							log.VEventf(ctx, 1, "starting split scattered %s in non-empty range %s", moved, reply.ScatteredSpan)
```

Comment:
not a big deal but any reason this is vmodule 1, when the same logline in the buffering adder is 3?

Reply:
made them both 1.